### ISSUE TITLE
RC Modeling Consulting and Assistance

### ISSUE THREAD

**Issue by Perference2025** [Giuleus957/trnpy#1]
Hello, PhD I'm opening an issue because it is probably the only way to contact you. I'm very glad to see that you are also working on the issue of building RC modeling. Could you please tell me if the issue regarding the state estimation of the RC modeling that you were inquiring about has been resolved? My concern is that when performing RC model identification, the fitting is good in the training set; however, once the validation set is used, the error significantly increases, the coupling parameters "drift" to an unreasonable range, and as a result, the subsequent MPC/PI control performance is also very poor (it is difficult to balance energy consumption and temperature simultaneously). Thank you for taking the time to consider my request. I look forward to your response.

Sincerely,
